### PR TITLE
Implementing project page layout

### PR DESCRIPTION
**Pull request by abrl91**
Issue Number: #3205 

## PR Details
Added a project page for wrapping the project sidebar and projects resources

## PR Checklist
- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/Project/ProjectSideBar.tsx [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
     <div className={CLASS_NAME}>
       <span className={`${CLASS_NAME}__label`}>Workspace</span>
       <WorkspaceSelector />
       <hr className={`${CLASS_NAME}__divider`} />
-      <ProjectList
-        projects={projectsList}
-        handleProjectChange={setNewProject}
-        workspaceId={currentWorkspace.id}
-      />
+      {ProjectList && (
```
Comment (muhsinkamil) [inline packages/amplication-client/src/Project/ProjectSideBar.tsx:32]: @abrl91 Did you mean to check `projectsList` :)

Comment (abrl91) [reply packages/amplication-client/src/Project/ProjectSideBar.tsx:32]: Not really, it was just for a test and it was left by mistake. Thank you

**Comment by arielweinberger** [comment]
@abrl91 Where do we stand with this?

**Comment by abrl91** [comment]
> @abrl91 Where do we stand with this?

@barshimi and I are working on that together. We have to fix some CSS issues

File: packages/amplication-client/src/Workspaces/hooks/workspace.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+export type TSetData = {
+  setCurrentWorkspace: {
+    token: string;
+  };
+};
+
+export type DType = {
+  createWorkspace: models.Workspace;
+};
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/hooks/workspace.ts:15]: I am not sure what we are using this for, but if the idea is to export it to an external file, I would expect to see more logical names

File: packages/amplication-client/src/Workspaces/queries/resourcesQuery.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { gql } from "@apollo/client";
+
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/queries/resourcesQuery.ts:2]: is this file for a single query or queries? should we call this resourceQueries?

File: packages/amplication-client/src/Workspaces/queries/workspaceQuery.ts [hunk 4]
```diff
@@ -27,3 +27,19 @@ export const SET_CURRENT_WORKSPACE = gql`
     }
   }
 `;
+
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/queries/workspaceQuery.ts:30]: should we call this workspaceQueries?

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
great work!!! see my comments

File: packages/amplication-client/src/Project/ProjectSideBar.scss [hunk 5]
```diff
@@ -3,17 +3,18 @@
...
     @include regular-12;
+    margin-bottom: 8px;
     color: var(--black80);
   }
 
   &__divider {
-    border: 1px solid var(--black90);
-    width: 100%;
+    border: 1px solid var(--black5);
+    width: 99%;
```
[the remark this answers is not in the document]
Comment (barshimi) [reply packages/amplication-client/src/Project/ProjectSideBar.scss:18]: it has an issue with width and padding. the divider is long

[the remark this answers is not in the document]
Comment (yuval-hazaz) [reply packages/amplication-client/src/Project/ProjectSideBar.scss:22]: can we think of a more accurate way to fix it?

**Pull request merged**